core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -3476,7 +3476,7 @@ select
       <![CDATA[
 LogicalProject(PRODUCT_ID=[$1])
   LogicalJoin(condition=[=($11, $1)], joinType=[left])
-    LogicalJoin(condition=[AND(=($1, $2), =($3, $0))], joinType=[left])
+    LogicalJoin(condition=[AND(=($1, $2), =($3, $4))], joinType=[left])
```

Comment:
To clarify these are only subquery examples that have the wrong output/need changing right.

Reply:
Correct